pkg/node/address.go

```diff
@@ -455,3 +451,13 @@ func SetIPsecKeyIdentity(id uint8) {
 func GetIPsecKeyIdentity() uint8 {
 	return ipsecKeyIdentity
 }
+
+// GetK8sNodeIPs returns k8s Node IP addr.
+func GetK8sNodeIP() net.IP {
+	return k8sNodeIP
+}
+
+// SetK8sNodeIP sets k8s Node IP addr.
+func SetK8sNodeIP(ip net.IP) {
+	k8sNodeIP = ip
```

Comment:
Do we need some kind of synchronization here?

Reply:
(Could/Should it be able to change over time?)

Reply:
For now noup, as getting the val happens after setting it.